docs/content/stable/api/ysql/ddl-behavior.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Behavior of DDL statements [YSQL]
+headerTitle: Behavior of DDL statements
+linkTitle: Behavior of DDL statements
+description: Explains how the behavior of DDL statements works in YugabyteDB YSQL and documents differences from Postgres behavior. [YSQL].
+menu:
+  stable_api:
+    identifier: ddl-behavior
+    parent: api-ysql
+    weight: 20
+type: docs
+---
+
+
+This section describes how DDL statements work in YSQL and documents the differences between YugabyteDB and PostgreSQL.
+
+## Concurrent DML during a DDL operation
+
+In YugabyteDB, DML is allowed to execute while a DDL statement modifies the schema that is accessed by the DML statement. For example, an `ALTER TABLE <table> .. ADD COLUMN` DDL statement may add a new column while a `SELECT * from <table>` executes concurrently on the same relation. In PostgreSQL, this is typically not allowed because such DDL statements take a table-level exclusive lock that prevents concurrent DML from executing. (Support for similar behavior in YugabyteDB is being tracked in [this github issue](https://github.com/yugabyte/yugabyte-db/issues/11571).)
```

Comment:
```suggestion
In YugabyteDB, DML is allowed to execute while a DDL statement modifies the schema that is accessed by the DML statement. For example, an `ALTER TABLE <table> .. ADD COLUMN` DDL statement may add a new column while a `SELECT * from <table>` executes concurrently on the same relation. In PostgreSQL, this is typically not allowed because such DDL statements take a table-level exclusive lock that prevents concurrent DML from executing. (Support for similar behavior in YugabyteDB is being tracked in issue {{<issue 11571>}}.)
```